docs/guide/architecture-styles/event-driven-content.md

```diff
@@ -41,6 +41,7 @@ In the logical diagram above, each type of consumer is shown as a single box. In
 
 - Guaranteed delivery. In some systems, especially in IoT scenarios, it's crucial to guarantee that events are delivered.
 - Processing events in order or exactly once. Each consumer type typically runs in multiple instances, for resiliency and scalability. This can create a challenge if the events must be processed in order (within a consumer type), or if the processing logic isn't idempotent.
+- Multiple events related to a single business process. [Workflow patterns](https://docs.particular.net/architecture/workflows/) can be used to reliably coordinate message flows across various services.
```

Comment:
@ckittel I've tried to keep the style of the first sentence aligned with the other bullet points. Does the following suggestion align better:

```suggestion
- Coordinating messages across services. Business processes often involve multiple services publishing and subscribing messages to achieve a consistent outcome across a whole system. [Workflow patterns](https://docs.particular.net/architecture/workflows/) can be used to reliably manage message flows across various services.
```